stages/org.osbuild.copy

```diff
@@ -20,6 +20,7 @@ from urllib.parse import urlparse, ParseResult
 
 import osbuild.api
 
+CAPABILITIES = ["CAP_SYS_ADMIN"]
```

Comment:
I think you mean `CAP_MAC_ADMIN`. Might also need `SYS_ADMIN` additionally, haven't checked.

Reply:
Added